tests/foreman/ui/test_user.py

```diff
@@ -639,6 +659,28 @@ def test_positive_update_email(self):
             self.user.validate_user(username, 'email', new_email)
 
     @tier1
+    def test_positive_update_description(self):
+        """Update Description in User
+
+        :id: f08ee305-0e0b-4df0-82d9-d10edcfa66c0
+
+        :expectedresults: User is updated successfully
+
+        :CaseImportance: Critical
+        """
+        username = gen_string('alpha')
+        description = gen_string('alpha')
+        with Session(self.browser) as session:
+            make_user(session, username=username, description=description)
+        for new_description in valid_data_list():
+            with self.subTest(new_description):
+                with Session(self.browser):
```

Comment:
No need in extra session (another login/logout), just continue using previous one.